packages/itwinui-css/backstop/tests/expandable-block.html

```diff
@@ -341,9 +372,9 @@ <h2>Borderless</h2>
     </section>
 
     <script>
-      document.querySelectorAll('.iui-header:not([aria-disabled="true"])').forEach((el) => {
+      document.querySelectorAll('.iui-expandable-header:not([aria-disabled="true"])').forEach((el) => {
         el.addEventListener('click', (event) => {
-          event.currentTarget.parentElement.classList.toggle('iui-expanded');
+          event.currentTarget.parentElement.classList.toggle('[data-iui-expanded="true"]');
```

Comment:
this code doesnt work. you need to set the attribute, not class.

something like this might work (untested):
```suggestion
        const prevExpanded = event.currentTarget.parentElement.dataset.iuiExpanded;
        event.currentTarget.parentElement.dataset.iuiExpanded = !prevExpanded;
```

another option is to use `setAttribute`